cms/djangoapps/contentstore/views/component.py

```diff
@@ -27,6 +27,12 @@
 from xmodule.modulestore.django import modulestore
 from xmodule.modulestore.exceptions import ItemNotFoundError
 
+try:
+    from lti_consumer import add_dynamic_components
```

Comment:
@jmaupetit  I think we need to add a comment here to recall that this lti_consumer endpoint targets our configurable xblock and not Open edX one because we are overriding it etc. ?